ch2driver/pytpcc/tpcc.py

```diff
@@ -478,7 +485,7 @@ def executorFunc(clientId, TAFlag, driverClass, schema, analyticalQueries, qDone
     ## Load Configuration file
     if args['config']:
         logging.debug("Loading configuration file '%s'" % args['config'])
-        cparser = SafeConfigParser()
+        cparser = ConfigParser()
```

Comment:
We need to add an import like `from configparser import ConfigParser` to make this work.

Out of curiosity, what was wrong with `SafeConfigParser`?